packages/components/src/VirtualizedList/CellTitle/CellTitle.scss

```diff
@@ -69,6 +69,8 @@ $tc-list-title-icon-size: $svg-lg-size !default;
 	visibility: visible;
 }
 
+:global(.tc-list-large-row.active):hover :global(.tc-list-title .tc-actions),
+:global(.ReactVirtualized__Table__row.active):hover :global(.tc-list-title .tc-actions),
```

Comment:
You make a difference between selected and active. It can be a good think if we can customize them differently.
For now it's the same color but, let's decouple them : 
* add a `$tc-list-row-active-hover-bg` in `../colors.scss`
* separate this style bloc from selected using the new color accordingly